### ISSUE TITLE
Are application users truly necessary?

### ISSUE THREAD

**Issue by ecolui** [MicrosoftDocs/powerapps-docs#5076]
I went through the process on this page and did not create an application user. Everything worked as expected.  I have a multi-tenant web app with delegated DataVerse permissions, and I tested this in a variety of different tenants.  In my oAuth access (using client secret), I pass the .default and offline_access scopes to get a long-lived refresh token.

Since everything seems to work well without an application user,  do we really need to go through the process of creating an Application User? 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c9e671be-3690-d8a8-51f8-29be7fa55516
* Version Independent ID: 28f644c9-c091-3bd5-0fad-dd4b347685a7
* Content: [Use multi-tenant server-to-server authentication (Microsoft Dataverse) - Power Apps](https://learn.microsoft.com/en-us/power-apps/developer/data-platform/use-multi-tenant-server-server-authentication#prepare-a-method-to-deploy-the-application-user)
* Content Source: [powerapps-docs/developer/data-platform/use-multi-tenant-server-server-authentication.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/developer/data-platform/use-multi-tenant-server-server-authentication.md)
* Service: **powerapps**
* Sub-service: **dataverse-developer**
* GitHub Login: @ritesp
* Microsoft Alias: **ritesp**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @ecolui. This issue has been assigned to the doc author for review.

**Comment by phecke** [comment]
@ecolui The product documentation states that for a server-to-server scenario, an Application User is needed for authentication. While you may have your implementation working, there is no telling under what conditions it will fail.

Even is your code is working, there is an issue with who owns the data changes which are logged in the audit history of each record. Typically, you want the Application User or the initiating user to be on record as owning the data changes, not your personal account.
 
It is best to follow the guidance in our docs.

**Comment by ecolui** [comment]
@phecke - Thank you for the feedback.  I see your point about the ownership of data changes and the audit history.  There's definitely value there as it pertains to creating an application user, and it's something that I really wanted to do. However, in one of my tests, I created an application user and activated my integration, but the integration seemed to still be associated with the delegated user who went through the oAuth flow in my app, not the application user I created.  I even deactivated the application user and the integration continued to work with no issues. 

When I set up the Dynamics CRM permissions for my application, the "Application Permissions" option was disabled, so I had to use the "Delegated Permissions" (see image below).

<img width="1251" alt="image" src="https://github.com/MicrosoftDocs/powerapps-docs/assets/5906265/ead2357d-33ea-4746-b60d-a0f9f558766b">

Two follow-up questions:
1) In your docs, is the server-to-server scenario you described synonymous to the "Application Permissions" option in the image above? Is an application user required only for the "Application Permissions" option, or is it also required for "Delegated Permissions"?
2) Why is the "Application Permissions" option disabled in Azure when configuring the Dynamics CRM permissions.

Thank you for your feedback!
